### ISSUE TITLE
Move to libraries.io release history for PyPI and CRAN.

### ISSUE THREAD

**Issue by isuruf** [regro/cf-scripts#143]
When subscribed to each project, it gives a nice xml. https://gist.github.com/isuruf/96a3c662939d78caf36975a1e5ca05ec

**Comment by CJ-Wright** [comment]
CRAN is using libraries.io currently, though PyPI could be moved over.

**Comment by isuruf** [comment]
What I mean is, you can get an xml of all newly released packages sorted by time published. When you use this, you don't have to make a one request per package. It's just one request.

**Comment by CJ-Wright** [comment]
Ah ok, how long is that xml though? My main worry is that we miss some things because the xml isn't long enough (or we don't ping fast enough).

**Comment by isuruf** [comment]
> Ah ok, how long is that xml though? 

It's paginated. 100 per page.

**Comment by justcalamari** [comment]
Solved via #274

**Comment by isuruf** [comment]
@CJ-Wright, is the slow down the first time you ran the code or was it in a subsequent run?

**Comment by CJ-Wright** [comment]
Both, we tried to run it a bunch of times. We finally got through the entire setup, but it took a long time.
Additionally because of the lazy loading of the data, each process worker was trying to run subscriptions itself.

I expected the loading of the data to be slow initially, but it seemed to keep hitting the LibrariesIO API more than I expected (as shown by the missing json). The log is here: https://circleci.com/gh/regro/circle_worker/537

We can try it again if you think it is worth it.
